src/theme/components/helperErrorText.ts

```diff
@@ -1,20 +1,31 @@
-interface HelperErrorTextBaseStyle {
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+import { defineStyle, StyleFunctionProps } from "@chakra-ui/system";
+
+interface HelperErrorTextBaseStyle extends StyleFunctionProps {
```

Comment:
This makes sense and I see why I didn't need this in the `Checkbox` PR, this is needed when you are passing props to the theme functions.